src/components/AudioPlayer/SeekButton.tsx

```diff
@@ -1,30 +1,76 @@
-import ForwardIcon from '../../../public/icons/forward_10.svg';
-import ReplayIcon from '../../../public/icons/replay_10.svg';
+import { useMemo } from 'react';
 
-import { triggerSeek } from './EventTriggers';
+import { useSelector } from 'react-redux';
+import useSWRImmutable from 'swr/immutable';
 
+import NextIcon from '../../../public/icons/skip_next.svg';
+import PrevIcon from '../../../public/icons/skip_previous.svg';
+
+import { triggerSetCurrentTime } from './EventTriggers';
+
+import { getChapterAudioData } from 'src/api';
 import Button, { ButtonShape, ButtonVariant } from 'src/components/dls/Button/Button';
-import { withStopPropagation } from 'src/utils/event';
+import { selectAudioData, selectReciter } from 'src/redux/slices/AudioPlayer/state';
+import { selectHighlightedLocation } from 'src/redux/slices/QuranReader/highlightedLocation';
+import { makeChapterAudioDataUrl } from 'src/utils/apiPaths';
+import { getVerseTimingByVerseKey } from 'src/utils/audio';
+import { getChapterData } from 'src/utils/chapter';
+import { makeVerseKey } from 'src/utils/verse';
 
 export enum SeekButtonType {
-  FastForward = 'fastForward',
-  Rewind = 'rewind',
+  NextAyah = 'nextAyah',
+  PrevAyah = 'prevAyah',
 }
 
 type SeekButtonProps = {
   type: SeekButtonType;
   isLoading: boolean;
 };
 const SeekButton = ({ type, isLoading }: SeekButtonProps) => {
+  const { highlightedChapter, highlightedVerse } = useSelector(selectHighlightedLocation);
```

Comment:
We shouldn't use highlighted here, we should use the more generic hook. No? I think that we called it activeVerse